src/feedforward/feedforward_arguments.jl

```diff
@@ -135,11 +136,12 @@ function _add_feedforward_arguments!(
     parameter_type = get_default_parameter_type(ff, T)
     add_parameters!(container, parameter_type, ff, model, devices)
     if get_slacks(ff)
-        add_feedforward_slack_variables!(
+        _add_feedforward_slack_variables!(
```

Comment:
We should test that the Slack FFs are working

Reply:
true

Reply:
tests added in [192dc3c](https://github.com/NREL-Sienna/PowerSimulations.jl/pull/1130/commits/192dc3c02a1fbc41631abf9d6f41fca946efd826)